tpp-app/src/services/LogSymptomsService.js

```diff
@@ -0,0 +1,69 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+import {FLOW_LEVEL} from './utils/constants'
+import { initializeEmptyYear, GETsymptomsForDate, isValidDate } from './utils/helpers'
+
+
+/**
+ * Posts the user's symptom data into the local storage for the given date.
+ * @param day number
```

Comment:
imo safer to specify that 1st day is 0

Reply:
in addition to the month (January = 1) thing

Reply:
Done! I changed it to this: `@param day number (1st day of month = 1)`